npm/pkg/dataplane/policies/policymanager_linux.go

```diff
@@ -152,17 +167,25 @@ func egressJumpSpecs(networkPolicy *NPMNetworkPolicy) []string {
 	return specs
 }
 
-// noflush add to chains impacted
-// TODO use array instead of ...
-func (pMgr *PolicyManager) creatorForNewNetworkPolicies(allChainNames []string, networkPolicies ...*NPMNetworkPolicy) *ioutil.FileCreator {
-	creator := pMgr.newCreatorWithChains(allChainNames)
+func (pMgr *PolicyManager) creatorForNewNetworkPolicies(policyChains []string, networkPolicies []*NPMNetworkPolicy) *ioutil.FileCreator {
+	creator := pMgr.newCreatorWithChains(policyChains)
+
+	// 1. Activate NPM if necessary
+	if pMgr.shouldActivate() {
```

Comment:
This might be little confusing, can we change the name to isFirstPolicy, that way we will know that for first policy alone we need these rules?

Also how will we manage duplicates of these rules on very first bootup ?

Reply:
updated to isFirstPolicy() and isLastPolicy()